app/assets/stylesheets/refills/_comments.scss

```diff
@@ -55,7 +55,7 @@
       margin-bottom: 0.5em;
     }
 
-    span {
+    span.timestamp {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3